web/src/features/AppVersionHistory/api/getVersions.tsx

```diff
@@ -5,17 +5,23 @@ import { useParams } from "react-router-dom";
 import { useSelectedApp } from "@features/App";
 import { useMetadata } from "@src/stores";
 import { useIsHelmManaged } from "@src/components/hooks";
+import { App, KotsParams, Metadata, Version } from "@types";
```

Comment:
nothing uses this hook yet but I saw some things I wanted to clean up. make it a typed component and got rid using default values to pass in imports.